[Footer] 고객문의 링크 연결, 회사 정보 컴포넌트 초안

ingbinsee opened the pull request:
## 🔍️ 이 PR을 통해 해결하려는 문제가 무엇인가요?

> 어떤 기능을 구현한건지, 이슈 대응이라면 어떤 이슈인지 PR이 열리게 된 계기와 목적을 Reviewer 들이 쉽게 이해할 수 있도록 적어 주세요
> 일감 백로그 링크나 다이어그램, 피그마를 첨부해도 좋아요

- Footer의 기능 및 레이아웃을 구현하였습니다.

## ✨ 이 PR에서 핵심적으로 변경된 사항은 무엇일까요?

> 문제를 해결하면서 주요하게 변경된 사항들을 적어 주세요

- 분기점(모바일, 태블릿, 데스크탑)에 따른 반응형 레이아웃 구현
- ContactUsSection, OfficeInfoSection 컴포넌트 제작
- ContactUsSection 클릭 시 Contact 페이지로 이동
- 회사 연락처 클릭 시 해당 번호로 통화 연결

## 🔖 핵심 변경 사항 외에 추가적으로 변경된 부분이 있나요?

> 없으면 "없음" 이라고 기재해 주세요

- 없음

## 🙏 Reviewer 분들이 이런 부분을 신경써서 봐 주시면 좋겠어요

> 개발 과정에서 다른 분들의 의견은 어떠한지 궁금했거나 크로스 체크가 필요하다고 느껴진 코드가 있다면 남겨주세요

- 없음

## 🩺 이 PR에서 테스트 혹은 검증이 필요한 부분이 있을까요?

> 테스트가 필요한 항목이나 테스트 코드가 추가되었다면 함께 적어주세요

- 없음

### 📌 PR 진행 시 이러한 점들을 참고해 주세요

- Reviewer 분들은 코드 리뷰 시 좋은 코드의 방향을 제시하되, 코드 수정을 강제하지 말아 주세요.
- Reviewer 분들은 좋은 코드를 발견한 경우, 칭찬과 격려를 아끼지 말아 주세요.
- Review는 특수한 케이스가 아니면 Reviewer로 지정된 시점 기준으로 3일 이내에 진행해 주세요.
- Comment 작성 시 Prefix로 P1, P2, P3 를 적어 주시면 Assignee가 보다 명확하게 Comment에 대해 대응할 수 있어요
  - P1 : 꼭 반영해 주세요 (Request Changes) - 이슈가 발생하거나 취약점이 발견되는 케이스 등
  - P2 : 반영을 적극적으로 고려해 주시면 좋을 것 같아요 (Comment)
  - P3 : 이런 방법도 있을 것 같아요~ 등의 사소한 의견입니다 (Chore)

#

---

## 📝 Assignee를 위한 CheckList

- [ ] To-Do Item

ingbinsee committed:
✨ [Feat] : Contact Us 링크 컴포넌트 완료

Footer에 Contact 페이지로 이동하는 섹션을 추가하였습니다.
+ 링크 클릭 시 Contact 페이지로 이동합니다.
+ 분기점(모바일, 태블릿, 데스크탑)에 따른 변화를 반영하였습니다.
+ module.css로 스타일을 분리하였습니다.

Related to: #44

ingbinsee committed:
✨ [Feat] : 회사 정보 컴포넌트 완료

Footer에 회사 정보 섹션을 추가하였습니다.
+ 분기점(모바일, 태블릿, 데스크탑)에 따른 변화를 반영하였습니다.
+ module.css로 스타일을 분리하였습니다.

Related to: #45

ryudg commented on src/components/Footer/OfficeInfoSection.jsx:
```diff
@@ -0,0 +1,22 @@
+function OfficeInfoSection() {
+  return (
+    <section className={styles.officeWrapper}>
+      <div className={`${styles.officeInfo} ${styles.officeText}`}>
+        <span>경기도 화성시 동탄첨단산업1로 51-9 M타워 1201호</span>
+        <div className="flex gap-open-gutter-mobile tablet:gap-open-gutter-desktop desktop:gap-open-gutter-desktop">
+          <Link to="tel:031-373-2612">T. 031-373-2612</Link>
+          <span>F. 031-377-2615</span>
+        </div>
+      </div>
+      <small className={`${styles.officeCopyright} ${styles.officeText}`}>
+        Copyright &copy;OpenSG, All rights reserved.
+      </small>
+    </section>
```
**P3**: `address` 태그 사용을 고려해 보세요

[MDN - `address`](https://developer.mozilla.org/ko/docs/Web/HTML/Element/address)

ingbinsee replied:
마크업 수정하였습니다.

kelvin-chihyun commented on src/components/Footer/ContactUsSection.jsx:
P3: section 태그를 주제별 섹션으로 분리하고자 한다면, 개인적으로는 section태그가 최상위에 위치하는 것이 어떨까 싶습니다.
그리고, h3 태그를 사용해주셨지만, section의 헤딩은 sr-only를 줘서, 바로 section 밑에 위치시키는게 좋아보입니다!

[참고 자료](https://codingeverybody.kr/html-section-%ED%83%9C%EA%B7%B8-%EC%98%AC%EB%B0%94%EB%A5%B8-%EC%9D%B4%ED%95%B4%EC%99%80-%EC%82%AC%EC%9A%A9-%EB%B0%A9%EB%B2%95/)

ingbinsee replied:
- Footer를 &lt;h2&gt;로 설정하고 Contact Us와 회사정보를 &lt;h3&gt;으로 설정하였습니다.
- &lt;section&gt;을 최상위에 위치하도록 하였습니다.
- Nesting 가능한 자식 태그를 부모 태그 아래로 두었습니다.

kelvin-chihyun reviewed (approved):
따로 남긴 의견 외에 이의 없습니다!

ingbinsee committed:
[Merge] : 원격 저장소와 로컬 저장소 동기화

ingbinsee committed:
♻️ [Refactor] : 마크업 수정

물리적 주소, 연락처, 팩스번호 등 연락처 정보에 대한 내용을 <div>에서 <adress>로 마크업을 수정하였습니다.

Related to: #46

ingbinsee committed:
♻️ [Refactor] : 리뷰 사항 반영

+ Footer를 <h2>로 설정하고 Contact Us와 회사정보를 <h3>으로 설정하였습니다.
+ <section>을 최상위에 위치하도록 하였습니다.
+ Nesting 가능한 자식 태그를 부모 태그 아래로 두었습니다.

Related to: #46

ingbinsee committed:
🤝 [Merge] : 원격 저장소와 로컬 저장소 동기화

ingbinsee committed:
🤝 [Merge] : 원격 저장소와 로컬 저장소 동기화
